src/OrchardCore/OrchardCore.ResourceManagement.Core/Razor/ResourceCdnHelperExtensions.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using OrchardCore;
+using OrchardCore.ResourceManagement;
+
+public static class ResourceCdnHelperExtensions
+{
+    /// <summary>
+    /// Prefixes the Cdn Base URL to the specified resource path.
+    /// </summary>
+    public static string ResourceUrl(this IOrchardHelper orchardHelper, string resourcePath, bool? appendVersion = null)
```

Comment:
Is this duplicated from somewhere, looks familiar

Reply:
This is not duplicate. This class was moved from from [OrchardCore.ResourceManagement](https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore/OrchardCore.ResourceManagement/Razor/ResourceCdnHelperExtensions.cs) project to drop the `OrchardCore.Abstractions` dependency from `OrchardCore.ResourceManagement`. So now, `OrchardCore.ResourceManagement` only depends on `OrchardCore.ResourceManagement.Core`